src/ICP.h

```diff
@@ -23,6 +24,9 @@
 
 class HttpRequest;
 
+class AccessLogEntry;
```

Comment:
please place these pre-defines in alphabetical order.

Reply:
Done.